README.md

```diff
@@ -15,3 +15,11 @@ Friday
 The Conjuring
 Enemy
 Nocturnal Animals
+
+Top five albums:
+Frank Ocean - Blonde
+Drake - More Life
+Travis Scott - Astroworld
+Wiz Khalifa - Kush & Orange Juice
+Mac Miller - KIDS
+Partynextdoor - Partynextdoor
```

Comment:
Partynextdoor???? He's trash. Change this lol

Reply:
@mjfaade10 this has been updated!